### ISSUE TITLE
Global Stereographic map from RGB grids (grdimage)

### ISSUE THREAD

**Issue by Esteban82** [GenericMappingTools/gmt#521]
Copy from http://gmt.soest.hawaii.edu/boards/1/topics/7836?r=8142#message-8142

I want to make a stereographic map from RGB grids for the whole earth (not center in the 
equator). I obtained the lower border cut. When I make a Mollweide map this doesn't happen.

**Full script that generated the error**

```
gmt grdimage -Rd -JW15c "Mundo_r.grd" "Mundo_g.grd" "Mundo_b.grd" -B0 > Globe_W.ps
gmt grdimage -Rd -JS-65/-30/90/15c "Mundo_r.grd" "Mundo_g.grd" "Mundo_b.grd" -B0 > Globe_S.ps



**System information**

* Operating system: Windows
* Version of GMT: 5.4.5

**Comment by PaulWessel** [comment]
So some rouge software package called "Mirone" [:-)] writes grids such as these:

> Mundo_b.grd: Title: Grid created by Mirone
> Mundo_b.grd: Command: 
> Mundo_b.grd: Remark: File written from Matlab
> Mundo_b.grd: Gridline node registration used [Geographic grid]
> Mundo_b.grd: Grid file format: nb = GMT netCDF format (8-bit integer), CF-1.7
> Mundo_b.grd: x_min: -179.95 x_max: 179.95 x_inc: 0.1 (6 min) name: longitude n_columns: 3600
> Mundo_b.grd: y_min: -84.9988712202 y_max: 85.0011287798 y_inc: 0.1 (6 min) name: latitude n_rows: 1701
> Mundo_b.grd: z_min: -128 z_max: 127 name: z [unknown]
> Mundo_b.grd: scale_factor: 1 add_offset: 128 packed z-range: [-256,-1]
> Mundo_b.grd: format: netCDF-4 chunk_size: 3600,1701 shuffle: on deflation_level: 5
> GEOGCS["WGS 84",
>     DATUM["WGS_1984",
>         SPHEROID["WGS 84",6378137,298.2572235604902,
>             AUTHORITY["EPSG","7030"]],
>         AUTHORITY["EPSG","6326"]],
>     PRIMEM["Greenwich",0],
>     UNIT["degree",0.0174532925199433],
>     AUTHORITY["EPSG","4326"]]

So it is not exactly repeating but claims it is grid node registered.  Yet it has all the hall marks of being a pixel grid.  We should be able to overcome such limitations but I suspect our code is tricked by the latitude range which is not "global".  Will have to see in debug probably.  It fails on a single grid as well:

`gmt grdimage -Rd -JS-65/-30/90/15c "Mundo_r.grd" > t.ps
`

**Comment by PaulWessel** [comment]
BTW, if I convert it to a proper pixel grid:

`gmt grd2xyz Mundo_r.grd | gmt xyz2grd -R-180/180/-85/85 -I0.1 -r -Gt.nc -V -fg
`

it still fails in the plot.  However, if I extend the range to ±90 then it works.

**Comment by PaulWessel** [comment]
Closing this issue as partially fixed by #543 and needing better up-stream grid preparation to ensure proper registration and domain settings.
